guides/source/active_record_basics.md

```diff
@@ -21,27 +21,24 @@ What is Active Record?
 ----------------------
 
 Active Record is the M in [MVC][] - the model - which is the layer of the system
```

Comment:
Yeah I see what you mean. I updated it to actually spell that out and include a reference to Active Model too. 

I think leaving the "M in MVC" part is useful. Especially for a beginner to connect the dots. We do say that Action Controll is the "C in MVC" in that guide. I'm leaning towards adding "V in MVC" to the Action View guide, that I'm currently editing..

Reply:
Makes sense to mention "V in MVC" in that guide.

I like the idea of connecting the dots more easily, we can totally keep that, just would like to find ways to disconnect a bit that C is only AR like I mentioned, because has been the common thought for a long time. :+1: